### PR TITLE
Adding barney-s to approvers

### PR DESCRIPTION
**Pull request by barney-s**
Adding barney-s to approvers to help with the project velocity. 
We are planning a re arch of the controller to bring it to latest kubebuilder version
Setup travis etc.

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -7,6 +7,7 @@ aliases:
     - prydonius
     - kow3ns
     - janetkuo
+    - barney-s
   apps-application-reviewers:
```
Comment (janetkuo) [inline OWNERS_ALIASES]: Do you want to add yourself as reviewers as well?

**Comment by janetkuo** [comment]
@barney-s you need to first request to be a member of kubernetes-sigs.

File: OWNERS_ALIASES (hunk 1 above)
Comment (barney-s) [reply OWNERS_ALIASES]: done

**Comment by barney-s** [comment]
Accepted request to join kubernetes-sigs org. Iam a member now.

**Comment by barney-s** [comment]
@mortent Would you lgtm this ?

**Comment by janetkuo** [comment]
/lgtm

I'll leave this to @kow3ns @mattfarina to approve.

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -34,3 +35,5 @@ aliases:
     - runseb
     - surajssd
     - kow3ns
+    - barney-s
+
```
Comment (prydonius) [inline OWNERS_ALIASES]: spurious new line?

Comment (barney-s) [reply OWNERS_ALIASES]: removed.

**Comment by janetkuo** [comment]
/lgtm
/approve

Approving based on the above approvals.

**Pull request merged**